pkg/apply/applier.go

```diff
@@ -159,13 +170,81 @@ func (a *Applier) readAndPrepareObjects() ([]*resource.Info, error) {
 		return nil, err
 	}
 
+	// Fetch the grouping label used for the current set of resources.
+	groupingLabel, err := prune.RetrieveInventoryLabel(inventoryObject.Object)
```

Comment:
Should this be `inventoryLabel` instead of `groupingLabel` ?

Reply:
fixed